src/amo/components/Link.js

```diff
@@ -10,6 +10,7 @@ import Icon from 'ui/components/Icon';
 
 export class LinkBase extends React.Component {
   static propTypes = {
+    blackIcon: PropTypes.bool,
```

Comment:
I was trying to avoid a situation where a caller could set both `external` and `externalDark` and end up with two icons, which is why I split it up into two props that do different things, as opposed to two props which each mean "it's an external link". I've changed it to use `externalDark` instead, and added some logic to avoid two icons.